### DISCUSSION TITLE
Draw.io "request entity too large" any way to incrase the request size?

### DISCUSSION THREAD

**Discussion by jan-runkel-zeitgleich** [requarks/wiki#3558] [Error / Bug Report] [answered]
**Describe the bug**
Draw.io in wikijs is awesome 👍 
If i add an image in Draw.io there will be Base64 encoded inside the Draw.io Data.
But the max size for a wikijs page is not enough for images. 

Is there a way to incrase the max page size?
![image](https://user-images.githubusercontent.com/64015392/93746746-effb3b00-fbf5-11ea-8727-917140b4fc7a.png)

**Comment by John222** [comment]
We experience the same issue.

We use right now the version: `2.5.268`

**Comment by NGPixel** [comment, marked as answer]
You can increase the body parser limit in your config.yml:

https://github.com/requarks/wiki/blob/main/config.sample.yml#L141-L147

**Reply by John222** [reply]
Thank you for the hint. This helped. We had to migrate to a newer version of wikijs. As the parameter were ignored in the mentioned version we had before.
